1_space_odyssey/js/mannuel-9000/farida-9000/app.js

```diff
@@ -0,0 +1,19 @@
+var isAlive = true;
+var name = "farida-9000";
+var hasJetPack = true;
+var suitColor = "pink";
+var eyes = 2;
+var eyeColor = "green";
+var hairColor = "red";
+var legs = 2;
+var arms = 2;
+var hands = 2;
+var ears = 2;
+var canWalk = true;
+var canRun = true;
+var canFly = true;
+var hasWeapon = true;
+var isInGoodMood = false;
+
+console.log(".-.-. Booting .-.-.");
+console.log(name, isAlive, hasJetPack, eyes, eyeColor, hairColor, legs, arms, hands, ears, canWalk, canFly, hasWeapon);
```

Comment:
please do exercise 2. as well.

its this one: https://github.com/mannuelf/feu1_programming_foundations/blob/master/1_space_odyssey/js/mannuel-9000/app.js#L111

![https://media.giphy.com/media/O8jYcO7IQHFWE/giphy-downsized.gif](https://media.giphy.com/media/O8jYcO7IQHFWE/giphy-downsized.gif)